tests/unit_tests/gui/test_main_window.py

```diff
@@ -500,13 +500,6 @@ def handle_add_dialog():
         experiments_panel.setCurrentIndex(1)
         current_tab = experiments_panel.currentWidget()
         assert current_tab.objectName() == "initialize_from_scratch_panel"
-        combo_box = get_child(current_tab, CaseSelector)
-
-        # Select "new_case"
-        current_index = 0
-        while combo_box.currentText().startswith("new_case"):
-            current_index += 1
-            combo_box.setCurrentIndex(current_index)
```

Comment:
There is no case named new case. So it didn't do anything.
It also says `# Select "new_case"` while actively doing the opposite. It would select the first thing that is not new_case.